app.py

```diff
@@ -78,45 +78,6 @@ def show_file_save_dlg(self):
         """
         self._tk_multi_workfiles.WorkFiles.show_file_save_dlg()
 
-    def _is_pyside_unstable(self):
```

Comment:
Previous version of this work we were under the impression certain versions of Nuke were having issues and not other DCCs. Turns out it was all versions and now all are stable, so removing the code.